scripting/get5.sp

```diff
@@ -1894,6 +1923,7 @@ bool FormatCvarString(ConVar cvar, char[] buffer, int len) {
   ReplaceString(team2Str, sizeof(team2Str), " ", "_");
 
   // MATCHTITLE must go first as it can contain other placeholders
+  ReplaceString(buffer, len, "{DATE}", formattedDate, false);
```

Comment:
Remove all case insensitivity for placeholders. I took it away in a PR on dev, as you can see in the calls below.